client/src/components/App/Carousel.jsx

```diff
@@ -37,4 +37,4 @@ Carousel.propTypes = {
   size: PropTypes.number.isRequired,
 };
 
-export default Carousel;
+export default Carousel;
```

Comment:
Looks like the new line at the end of the file was deleted. Would you mind adding it back in to avoid adding extraneous changes on the merge?